cluster/plugin/local/plugin.go

```diff
@@ -99,7 +101,7 @@ func removeAgent(ctx context.Context, c *client.Client, cid string) error {
 	return c.ContainerRemove(ctx, cid, types.ContainerRemoveOptions{Force: true})
 }
 
-// RunAgent runs the ampagent image to init (action ="") or destroy (action="uninstall")
+// RunAgent runs the ampagent image to init (action ="install") or destroy (action="uninstall")
```

Comment:
You've done a good job here making do with the current agent implementation, but it's a flaw in the agent that the cmd part isn't more separated from the package and it's something I want to fix. You should be able to just import the package to use here, not have to run another process just to pass the commands.